src/main/java/com/manywho/services/azure/oauth/AzureConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package com.manywho.services.azure.oauth;
+
+import com.manywho.sdk.services.utils.Environment;
+import com.manywho.services.azure.ServiceConfiguration;
+
+import javax.inject.Inject;
+
+public class AzureConfiguration {
+    public static final String REDIRECT_URI = "https://flow.manywho.com/api/run/1/oauth2";
+    public static final String GRAPH_RESOURCE = "00000003-0000-0000-c000-000000000000";
+    public static final String DIRECTORY_NAME = "azure-manywho";
+
+    private final ServiceConfiguration serviceConfiguration;
+
+    @Inject
+    AzureConfiguration(ServiceConfiguration serviceConfiguration) {
+        this.serviceConfiguration = serviceConfiguration;
+    }
+
+    /**
+     * If tenant is not empty returns the oauth2 entry point v1:
```

Comment:
Comments do not seem to match logic

Reply:
Thank you,
I have removed the v1/v2 from the format of the entry point comments, as that would be confusing, and also added a link to the official docs.